src/components/Navbar/NavBar.tsx

```diff
@@ -38,6 +40,33 @@ const NavBar = (props: Props) => {
     }
   };
 
+  const handleDropdownOpen = () => {
+    setDropdownOpen(!dropdownOpen);
+  };
+
+  const dropdownOptions = [
+    { url: "/merchants", translation: "donate", external: false },
+    { url: "/gift-a-meal-home", translation: "gift-a-meal", external: false },
+    { url: "https://stickylocals.com/scl", translation: "sticky-locals-x-scl", external: true  },
+    { url: "https://www.friendofafriend.studio/shop", translation: "friendofafriend-studioxscl", external: true }
+  ]
+ 
+  const Drop = () => {
+    return (
+    dropdownOptions.map(option => {
+    const { url, translation, external } = option; 
+      return external ? (
+         <DropdownItem href={url}>
+           { t(`navBar.header.waystodonate.${translation}`) }
+         </DropdownItem>
+         ) : (
+         <DropdownItemLink to={url}>
+           { t(`navBar.header.waystodonate.${translation}`) }
+         </DropdownItemLink>
+          )
+    }))
+  }
```

Comment:
nit: You don't really need to wrap this in a function, assigning it to a variable is fine. Also the variable name should be camelCase.

Not a blocking issue - go ahead merge the PR. 

Unrelated but for clean code, you can just do this
`const drop = () =>
    dropdownOptions.map(option => {
... })
`